### PR TITLE
doc: profiling remote compute nodes

### PR DESCRIPTION
**Pull request by yuhao-su**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ wget https://github.com/risingwavelabs/risingwave/releases/download/v0.1.15/risi
 # Unzip the binary
 tar xvf risingwave-v0.1.15-x86_64-unknown-linux.tar.gz
 # Start RisingWave in single-binary playground mode
-./risingwave playground
+./risedev playground
```
Comment (TennyZhuang) [inline README.md:29]: It's risingwave I guess (

Comment (BugenZhao) [reply README.md:29]: +1. We don't have risedev in the release.

File: docs/cpu-profiling.md [hunk 2]
```diff
@@ -13,3 +14,12 @@ Generate flamegraph:
...
 ```
+
+## Profiling remote compute nodes
+You can profile remote compute nodes from a local machine by simply type the following command.
+```shell
+./risedev ctl profile --sleep [seconds]
+```
+All compute nodes will be profile for a given `seconds` time and generated flame graph will be transferred to your local machine.
+
+Note: To profile our compute nodes remotely, please make sure all remote nodes have a public ip address accessible from your local machine (The one)
```
Comment (BugenZhao) [inline docs/cpu-profiling.md:25]:
```suggestion
Note: To profile compute nodes remotely, please make sure all remote nodes have a public IP address accessible from your local machine (The one 😴?)
```

File: docs/cpu-profiling.md [hunk 3]
```diff
@@ -13,3 +14,12 @@ Generate flamegraph:
...
 ```
+
+## Profiling remote compute nodes
+You can profile remote compute nodes from a local machine by simply type the following command.
+```shell
+./risedev ctl profile --sleep [seconds]
+```
+All compute nodes will be profile for a given `seconds` time and generated flame graph will be transferred to your local machine.
+
+Note: To profile compute nodes remotely, please make sure all remote nodes have a public IP address accessible from your local machine (The one you are running `./risedev`)
```
Comment (BugenZhao) [inline docs/cpu-profiling.md:25]:
```suggestion
Note: To profile compute nodes remotely, please make sure all remote nodes have a public IP address accessible from your local machine (where you are running `risedev`).
```

**Review by BugenZhao — APPROVED** [review]

**Pull request merged**